src/components/registreringsInfo.tsx

```diff
@@ -0,0 +1,143 @@
+        (item) => item.sporsmalId === 'utdanning'
+    );
+    const UtdanningSvar: StringOrNothing = regDataUtdanning?.svar;
+
+    const regDataUtdanningGodkjent = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanningGodkjent'
+    );
+    const UtdanningGodkjentSvar: StringOrNothing = regDataUtdanningGodkjent?.svar;
+
+    const regDataUtdanningBestatt = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'utdanningBestatt'
+    );
+    const UtdanningBestattSvar: StringOrNothing = regDataUtdanningBestatt?.svar;
+
+    const regDataHelse = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'helseHinder'
+    );
+    const HelseSvar: StringOrNothing = regDataHelse?.svar;
+
+    const regDataAnnet = registrering?.registrering?.teksterForBesvarelse.find(
+        (item) => item.sporsmalId === 'andreForhold'
+    );
+    const AnnetSvar: StringOrNothing = regDataAnnet?.svar;
+
+    const regIdNavn = registrertAvID + ' ' + registrertAvNavn;
+    const regDato = 'Registrert: ' + formaterDato(datoRegistrert);
+    const regAv = 'Registrert av: ' + registrertAvIdent + ', ' + regIdNavn;
+
+    const regValues = [`${regDato}`, `${regAv}`];
+
+    const brukerRegistrering = registrering?.registrering;
+    const type = registrering?.type;
+
+    return (
+        <Panel border className="info_panel">
+            <Heading spacing level="2" size="large">
+                Registering
+            </Heading>
+            <DobbeltInformasjon header={regAv ? regAv : EMDASH} values={regValues ? regValues : [EMDASH]} />
+            <span className="info_container">
+                <EnkeltInformasjon header="Hvorfor registrerer du deg?" value={hvorforSvar ? hvorforSvar : EMDASH} />
+                <EnkeltInformasjon header="Din siste jobb" value={sisteStillingSvar ? sisteStillingSvar : EMDASH} />
+                <EnkeltInformasjon
+                    header="Hva er din høyeste fullførte utdanning?"
+                    value={UtdanningSvar ? UtdanningSvar : EMDASH}
+                />
+                <EnkeltInformasjon
+                    header="Er utdanningen din godkjent i Norge?"
+                    value={UtdanningGodkjentSvar ? UtdanningGodkjentSvar : EMDASH}
+                />
+                <EnkeltInformasjon
+                    header="Er utdanningen din bestått?"
+                    value={UtdanningBestattSvar ? UtdanningBestattSvar : EMDASH}
+                />
+                <EnkeltInformasjon
+                    header="Trenger du oppfølging i forbindelse med helseutfordringer?"
+                    value={HelseSvar ? HelseSvar : EMDASH}
+                />
+                <EnkeltInformasjon
+                    header="Trenger du oppfølging i forbindelse med andre utfordringer?"
+                    value={AnnetSvar ? AnnetSvar : EMDASH}
+                />
+                <br />
```

Comment:
Det er ikke god praksis å bruke `<br>`-elementer til å lage layout, fordi `<br>` ikke er et semantisk element. Se for eksempel: [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns) og [HTML5-specen](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-br-element.html)